src/components/Login/Logout.jsx

```diff
@@ -27,7 +28,22 @@ const Logout = () => {
         <label id="labelLogout" htmlFor="btnLogout">
           Click the following logout button to log out of your pod:{' '}
         </label>
-        <LogoutButton onLogout={handleLogout} />
+        <button type="submit" onClick={() => setShowConfirmation(true)}>
+          Logout
+        </button>
+        {showConfirmation ? (
+          <dialog open>
+            <p>Do you want to log out now?</p>
+            <div>
```

Comment:
On Line 37, can do a little inline-styling with style attribute with display flex, justify content center and a gap to center the buttons in one row

Reply:
Updated -- that definitely looks better!